vignettes/replication.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Replication by `rdlearn`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{my-vignette}
```

Comment:
When I build this vignetted via `devtools::build_vignettes()` I got the following warning: 

> Warning: The vignette title specified in \VignetteIndexEntry{} is different from the title in the YAML metadata. The former is "my-vignette", and the latter is "Replication by `rdlearn`". If that is intentional, you may set options(rmarkdown.html_vignette.check_title = FALSE) to suppress this check.